File: sct_source/SourceCode/FE_SoCongThuong/Angular/src/app/modules/app-pages/commune-electricity-management-page/_models/commune-electricity-management.model.ts
import { BaseModel } from '../../../../_metronic/shared/crud-table/models/base.model';

export interface CommuneElectricityManagementModel extends BaseModel {
  communeElectricityManagementId : string;
  stageId : string;
  districtId: string;
  communeId: string;
  content41Start: boolean;
  content42Start: boolean;
  target4Start: boolean;
  content41End: boolean;
  content42End: boolean;
  target4End: boolean;
  note: string;
}

File: sct_source/SourceCode/API_SoCongThuong/Models/CommuneElectricityManagementModel.cs
namespace API_SoCongThuong.Models
{
    public class CommuneElectricityManagementModel
    {
        public Guid CommuneElectricityManagementId { get; set; }
        public Guid StageId { get; set; }
        public string? StageName { get; set; }
        public Guid DistrictId { get; set; }
        public string? DistrictName { get; set; }
        public Guid CommuneId { get; set; }
        public string? CommuneName { get; set; }
        public bool Content41Start { get; set; }
        public bool Content42Start { get; set; }
        public bool Target4Start { get; set; }
        public bool Content41End { get; set; }
        public bool Content42End { get; set; }
        public bool Target4End { get; set; }
        public string? Note { get; set; }
    }
}